core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1018,34 +1024,95 @@ public ExecResult execInContainer(String... command)
      * {@inheritDoc}
      */
     @Override
-    public void copyFileToContainer(MountableFile mountableLocalFile, String containerPath) {
+    @SneakyThrows(IOException.class)
+    public void copyFileToContainer(MountableFile mountableFile, String containerPath) {
```

Comment:
both functions do almost exactly the same. Can those two be merged? (Can't you just call the other`copyFileToContainer`function? `MountableFile` implements `Transferable`0

Reply:
👍